docs/contributing/dev-setup/runtests.rst

```diff
@@ -0,0 +1,16 @@
+.. _runtests:
+
+Run the Tests
+=============
+
+To run the tests provided by a specific plugin, start the django development server in the development vm with `pulp-manager runserver`.
+You need to create a *pulp-smash* config file with `pulp-smash config create` (this is ony needed once).
+Then on a second shell, you can perform the tests via `pulp-manager test <plugin_name>`.
```

Comment:
Pulp-smash is used to write functional tests. Perhaps add a link to the documentation itself.